front/app/components/IdeaCard/CardImage/index.tsx

```diff
@@ -24,11 +24,6 @@ const IdeaCardImageWrapper = styled.div<{ $cardInnerHeight: string }>`
     width: 100%;
     margin-bottom: 18px;
   `}
-
-  &.hasFilterSidebar {
-    width: 100%;
-    margin-bottom: 18px;
-  }
 `;
```

Comment:
Removing a number of occurrences of this styling since [we've decided to go for the 1 column layout](https://go-vocal.slack.com/archives/C07PEAULUNS/p1732624917946689?thread_ts=1732610211.533219&cid=C07PEAULUNS) instead now :)